package.json

```diff
@@ -46,8 +46,8 @@
     "sinon": "^10.0.1"
   },
   "dependencies": {
-    "@adobe/aio-lib-files": "^2.2.0",
-    "@adobe/asset-compute-commons": "^1.1.5",
+    "@adobe/aio-lib-files": "^2.2.1",
```

Comment:
What is/are the bugfix(es)?

Reply:
release: https://github.com/adobe/aio-lib-files/releases/tag/2.2.1

Updated deprecated dep @hapi/joi to joi (#98) 137e6d7 <- should be fine
Used default crypto (#99) 6c46fff <- should also be fine
Fixed BYO custom access policy flow (#97) 324adfa <- this one looks interesting. I think we ran into issues when using a custom container since we had custom policies set up (remember they just removed our policies and broke our containers without us knowing/realizing). But anyways, it is good they added this error handling so it wouldn't mess up a custom container with policies set. And we use their default containers so we shouldn't have this issue. I will wait for ITs to run to confirm.
Added daily npm install and test workflow (#96) d34c5a1 <- I would hope this doesn't break anything
Updated README.md a55350f / e932bbe <- I would hope this doesn't break anything

Reply:
I can remove this if you think it's too risky

Reply:
Just asking. We can go ahead.